cmd/prom2json/main.go

```diff
@@ -26,9 +27,12 @@ import (
 	"github.com/prometheus/prom2json"
 )
 
+const DefaultTimeout = 10 * time.Second
```

Comment:
I'm pretty sure this can be `defaultTimeout`, it doesn't need to be exported.

Reply:
In different news, I think the default behavior should be "no timeout" (as it was before this change).